exampleSite/config/_default/languages.en.toml

```diff
@@ -22,4 +22,5 @@ copyright = "&copy; 2023 Congo contributors"
     { facebook = "https://facebook.com/" },
     { linkedin = "https://linkedin.com/" },
     { youtube = "https://youtube.com/" },
+    { google-scholar = "https://scholar.google.com/"},
```

Comment:
Including this is useful to validate the PR but I'm not keen on adding Google Scholar to the example site. Please remove this from the config.